hack/olm-registry/olm-artifacts-template.yaml

```
@@ -546,14 +547,16 @@ objects:
               command:
               - /bin/audit-exporter
               - --log-file=/var/log/osd-audit/audit.log
+              - --tls-cert-file=/certs/tls.crt
+              - --tls-private-key-file=/certs/tls.key
               resources:
                 requests:
                   cpu: 10m
                   memory: 32Mi
                 limits:
                   cpu: 50m
                   memory: 128Mi
```

Can we bump these up a little, just to be safe? (metrics aren't enabled at all in prod currently) 

Maybe make the requests & limits the same to ensure the pods get a QoS class of Guaranteed

```suggestion
              resources:
                requests:
                  cpu: 100m
                  memory: 256Mi
                limits:
                  cpu: 100m
                  memory: 256Mi
```

---

Sounds good.